Makefile

```diff
@@ -1,6 +1,7 @@
 # Minimal makefile for Sphinx documentation
 #
 
+
```

Comment:
I don't see which typo this is supposed to fix. I only see white space changes. Can you please elaborate?

Reply:
The user in question has first tried to introduce curl script download into the build process in other repos and is later jsut spamming mostly empty line additons to various repositories